packages/xo-collection/src/collection.test.js

```diff
@@ -0,0 +1,356 @@
+import fromEvent from 'promise-toolbox/fromEvent'
+import forEach from 'lodash/forEach.js'
+
+import assert from 'assert'
+import { test, describe, beforeEach } from 'node:test'
```

Comment:
> It's not a conversion if you are not removing the old tests ;)

I just leave them for the time of the review, so that the reviewers can see the changes, as explained in the PR description. I will then delete them of course. :-)

Reply:
IMHO, it does not make things easier, I can see the removed files just fine on GitHub.